app/ide-desktop/lib/dashboard/src/components/Devtools/EnsoDevtools.tsx

```diff
@@ -92,6 +100,56 @@ export function PaywallDevtools(props: PaywallDevtoolsProps) {
 
             <ariaComponents.Separator orientation="horizontal" className="my-3" />
 
+            <ariaComponents.Text variant="subtitle">
+              {getText('paywallDevtoolsPlanSelectSubtitle')}
+            </ariaComponents.Text>
+
+            <ariaComponents.Form
+              gap="small"
+              schema={schema => schema.object({ plan: schema.string() })}
+              defaultValues={{ plan: session.user.plan ?? 'free' }}
+            >
+              {({ form }) => (
+                <>
+                  <ariaComponents.RadioGroup
```

Comment:
i think i would prefer a `ListBox` but i guess this works

Reply:
i suppose if the listbox is full width then it'd look a little weird

Reply:
radio works well when we have less than 5 options (approximately). 
And anyway this is for us, and only for dev mode.